### PR TITLE
Added vocabs used in W3C test cases and reports

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: ontologies/_index.nq [hunk 1]
```diff
@@ -67,6 +74,11 @@
 <https://prefix.zazuko.com/duv:> <http://www.w3.org/2000/01/rdf-schema#isDefinedBy> <http://www.w3.org/ns/duv#> .
 <https://prefix.zazuko.com/duv:> <http://www.w3.org/ns/rdfa#prefix> "duv" .
 <https://prefix.zazuko.com/duv:> <http://www.w3.org/ns/rdfa#uri> <http://www.w3.org/ns/duv#> .
+<https://prefix.zazuko.com/earl:> <http://dbpedia.org/ontology/filename> "ontologies/earl.nq" .
+<https://prefix.zazuko.com/earl:> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/rdfa#PrefixMapping> .
+<https://prefix.zazuko.com/earl:> <http://www.w3.org/2000/01/rdf-schema#isDefinedBy> <https://www.w3.org/ns/earl#> .
+<https://prefix.zazuko.com/earl:> <http://www.w3.org/ns/rdfa#prefix> "earl" .
+<https://prefix.zazuko.com/earl:> <http://www.w3.org/ns/rdfa#uri> <https://www.w3.org/ns/earl#> .
```
Comment (tpluscode) [inline ontologies/_index.nq:93]: RDF/XML fails to parse `owl:Ontology` and I think that's why there is no `dcterms:description`. Add manually?

File: README.md [hunk 2]
```diff
@@ -259,7 +259,7 @@ in the [DBpedia SPARQL endpoint](http://dbpedia.org/sparql?nsdecl) or other popu
 
 ### Steps to add a prefix
 
-1. Add an entry in [`src/lib/prefixes.ts`](src/lib/prefixes.ts)
+1. Add an entry in [`src/lib/prefixes.ts`](src/prefixes.ts)
```
Comment (vhf) [inline README.md]:
```suggestion
1. Add an entry in [`src/prefixes.ts`](src/prefixes.ts)
```

File: ontologies/_index.nq (hunk 1 above)
Comment (vhf) [reply ontologies/_index.nq:93]: I think it's getting parsed, here's what I get for description and label:

```
<https://www.w3.org/ns/earl> <http://www.w3.org/2000/01/rdf-schema#comment> "Formal schema of the Evaluation and Report Language (EARL) 1.0"@en <https://www.w3.org/ns/earl#> .
<https://www.w3.org/ns/earl> <http://www.w3.org/2000/01/rdf-schema#label> "Evaluation and Report Language (EARL) 1.0 Schema"@en <https://www.w3.org/ns/earl#> .
```

the reason it's not considered for the index is that the subject is `<https://www.w3.org/ns/earl>` when the vocab base IRI is `<https://www.w3.org/ns/earl#>`. I'll try a simple workaround, trimming trailing `#` and `/` for title and description

File: fetch.ts [hunk 3]
```diff
@@ -121,39 +123,52 @@ function firstVal (dataset: DatasetExt) {
...
+    return potentialValues[0]
+  }
+  return ''
 }
 
 function generateIndex (subject: NamedNode, mappings: any, dataset: DatasetExt) {
   const vocabUri = rdf.namedNode(mappings.uri)
-  const filteredDataset = dataset.match(vocabUri)
+  let filteredDataset = dataset.match(vocabUri)
+  if (vocabUri.value.endsWith('/') || vocabUri.value.endsWith('#')) {
```
Comment (vhf) [inline fetch.ts:155]: it did the trick and incidentally also generated many more missing titles and descriptions for other ontologies :)

**Pull request merged**